src/main/java/com/uploadcare/api/Client.java

```diff
@@ -252,26 +571,126 @@ public void close() {
     }
 
     /**
+     * @deprecated Use {@link #copyFileLocalStorage(String, Boolean, Boolean)}
+     * or {@link #copyFileRemoteStorage(String, String, Boolean, String)} instead.
+     *
      * @param fileId  Resource UUID
      * @param storage Target storage name
      * @return An object containing the results of the copy request
      */
+    @Deprecated
     public CopyFileData copyFile(String fileId, String storage) {
```

Comment:
Do I understand correctly, that fileId can also be a URL here as it's being passed to API as is?

Reply:
No it's not supporting CDN URL, only UUID of a file is supported.

Reply:
I believe that I see that a string is being passed over the network. It can contain both UUID or URL. This is supported by API, and should be supported by the lib.

Am I incorrect? E.g. you can't pass an Uploadcare CDN URL in `fileId` param?

Reply:
@dmitry-mukhin you are right, Solved in 79b3a804. I've added possibility to provide "CDN URL" as copy source in all copy methods local/external, updated comments, and Updated the returned result to contain the new copied instance of the file/url ready to be used according to latest docs. Tested.